certbot-apache/certbot_apache/constants.py

```diff
@@ -62,9 +62,9 @@
     vhost_files="*.conf",
     logs_root="/var/log/apache2",
     version_cmd=['/usr/sbin/apache2', '-v'],
-    define_cmd=['apache2ctl', 'virtualhosts'],
-    restart_cmd=['apache2ctl', 'graceful'],
-    conftest_cmd=['apache2ctl', 'configtest'],
+    define_cmd=['/usr/sbin/apache2', '-S'],
```

Comment:
@Jarodiv OpenRC is still Gentoos default, so I would prefer the default for `certbot` also to be OpenRC compatible. Until `certbot` has a system to detect the init system used, the Gentoo maintainer might add a use flag in combination with a patch for systemd users.